ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/flow-trackings.j2

```diff
@@ -0,0 +1,43 @@
+{% for flow_tracking in flow_trackings | arista.avd.default([]) %}
+!
+flow tracking {{ flow_tracking.type }}
+{%     if flow_tracking.sample is arista.avd.defined %}
+   sample {{ flow_tracking.sample }}
+{%     endif %}
+{%     for tracker in flow_tracking.trackers %}
+   tracker {{ tracker.name }}
+{%         if tracker.record_export.inactive_timeout is arista.avd.defined %}
+      record export on inactive timeout {{ tracker.record_export.inactive_timeout }}
+{%         endif %}
+{%         if tracker.record_export.interval is arista.avd.defined %}
+      record export on interval {{ tracker.record_export.interval }}
```

Comment:
I don't think we should necessarily endeavour to replicate 1:1 the CLI but if this is the rule I can indeed change the model as you describe.

The only time it could hurt us would be if some other option is added under record_export with its own inactive_timeout.

Happy to change it either way the consensus goes :)

Reply:
maybe just do `on_inactive_timeout` and `on_interval` to keep everything consistent. Also those key names are human readable and make some sense.

Reply:
Addressed in 58a284a